src/common/ChannelChatters.cpp

```diff
@@ -64,9 +64,31 @@ void ChannelChatters::addPartedUser(const QString &user)
         });
     }
 }
+
+const QColor ChannelChatters::getUserColor(const QString &user)
+{
+    const auto chatterColors = this->chatterColors_.access();
+
+    const auto search = chatterColors->find(user.toLower());
+    if (search == chatterColors->end())
+    {
+        return QColor(MessageColor::Text);
+    }
+    else
+    {
+        return search->second;
+    }
+}
+
```

Comment:
This function could be moved right above ChannelChatters::setUserColor to keep things in logical order.